tools/hilbert.py

```diff
@@ -0,0 +1,589 @@
+#!/usr/bin/env python2
```

Comment:
I would like to be able to actually call the `hilbert` command instead of calling
```
python[3] tools/hilbert.py
```